site/content/guide/index.md

```diff
@@ -86,6 +86,105 @@ Below is a screenshot of how the final website will look like:
 > [!tip]
 > Read full tutorial [[edit-a-website-locally|here!]]
 
+### Tutorial 3: Collaborating with others on your website project
+
+In this tutorial, we will guide you through collaborating with others on the same website project using GitHub. 
+
+By the end of this tutorial, you will:
+
+- Understand what a branch is and how to create one.
+- Understand what a pull request (PR) is and how to create one.
+- Learn how to review and merge a PR.
+- Know how to resolve conflicts if they arise.
+- Collaborate with others using GitHub, following best practices.
+
+#### Create a new branch
+
+- Navigate to your site's repository on GitHub
+- Click on the 'Branch' dropdown menu, type a new branch name and press Enter
+- Yay! You've created a new branch! 🎉
+
+#### Now, let's make changes in the new branch
+
+- Switch to the new branch in GitHub Desktop
+- Open the `/content` folder of the repository in Obsidian
+- Edit the `about` page
+- Commit the changes in GitHub Desktop app and push them to the remote repository on GitHub
+
+#### Create a pull request (PR)
+
+- Go back to your site's repository on GitHub
+- Click on 'New pull request'
+- Select your branch from the dropdown menu
+- Write a brief description of your changes, then click on 'Create pull request'
+- Yay! Your changes are ready to be reviewed! 🎉
+
+#### Review and merge a pull request
+
+- Navigate to the 'Pull requests' tab in your repository
+- Open the newly created pull request
+- Review the changes, add comments if necessary
+- If everything is in order, click 'Merge pull request', then 'Confirm merge'
+- Yay! You've merged your changes into the main branch! 🎉
+
+#### Resolving conflicts
+
+- In case of conflicts between your changes and those on the main branch, GitHub will alert you
+- Follow the on-screen instructions to resolve the conflicts and merge your changes
+
+> [!tip]
+> Read full tutorial [[collaborate-on-website-project|here!]] (TBD)
+
+### Tutorial 4: Customising your website locally and previewing your changes locally
+
+In this tutorial, we will dive into the more technical aspects of website customisation, but this time, everything will be done locally. You'll learn how to preview your site on your own machine before pushing changes to the live site, ensuring everything looks and works exactly as you want.
+
+By the end of this tutorial, you will:
+
+- Understand how to preview your site locally.
+- Know how to change your website's title and description.
+- Learn how to customise your website's appearance using Tailwind themes.
+- Understand how to configure the navbar, navigation links, and logo.
+- Learn how to integrate Google Analytics into your website.
+- Be aware of additional customisation options for deeper customisation.
+
+#### Previewing the site locally
+
+- Navigate to your website's repository directory on your computer using command line
+- Install the site's dependencies
```

Comment:
what does this mean? 
if the individual will know, ignore this comment

Reply:
It will be explained in the tutorial itself